lib/models.js

```diff
@@ -58,6 +58,16 @@ export class Path {
         return this.stat ? !this.stat.isDirectory() : null;
     }
 
+    isVisible() {
+      ignoredFiles = config.get('ignoredFiles');
+      if (!ignoredFiles) return true;
+      ignoredExtensions = ignoredFiles.split(',');
+      for( i in ignoredExtensions) {
+        if(this.fragment.endsWith(ignoredExtensions[i])) return false;
```

Comment:
I'm a little torn here, because this isn't strictly extension matching, it's suffix matching. Which is fine, but then the natural next step is to add prefix matching, etc. Maybe a list of regexes to check against the file?

Also, it might be more performant to add this ignoring logic to the `matchingPaths` method instead of adding `isVisible`, which would avoid creating a new `Path` object for paths that aren't going to be displayed.

Reply:
I applied these suggestions in the last commit: using globs (which I find more natural than regexp for filename); moved to the `matchingPaths` method; compile one and cache regexps for performances.